src/components/atoms/UserAvatar/UserAvatar.tsx

```diff
@@ -76,7 +89,16 @@ export const UserAvatar: React.FC<UserAvatarProps> = ({
         alt={`${userDisplayName}'s avatar`}
         onClick={onClick}
       />
-      {showStatus && <span className={statusIndicatorClasses} />}
+      {/*
+        'isStatusEnabledForVenue' checks if the user status is enabled from the venue config.
+        'showStatus' is used to render this conditionally only in some of the screens.
+      */}
+      {isStatusEnabledForVenue && showStatus && isOnline && (
```

Comment:
can we have a boolean const for `isStatusEnabledForVenue && showStatus && isOnline` and use it here?

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1532/commits/0c1f9f07a5696848014d78ad2604de3c1ed2d194